libs/ui/packages/native/.storybook/main.js

```diff
@@ -9,4 +9,7 @@ module.exports = {
     // "@storybook/addon-ondevice-backgrounds",
     "@storybook/addon-ondevice-notes",
   ],
+  typescript: {
+    reactDocgen: false, // FIXME: this is disabled for now due to incompatibilities with TS 5. re-enable when upgrading storybook.
```

Comment:
@aboissiere-ledger we will need to see what to do with storybook in future / have a team on it.